back/db/views/analytics_fact_participations_v06.sql

```diff
@@ -0,0 +1,29 @@
+-- Analytics view for all types of participation based on the activities table
+
+SELECT
+    a.id,
+    a.user_id AS dimension_user_id,
+    COALESCE(a.user_id, a.id) AS participant_id,
+    a.project_id AS dimension_project_id,
+    a.acted_at::DATE AS dimension_date_created_id,
+    a.item_type AS item_type,
+    a.action AS action_type,
+    a.payload->>'reactable_type' AS reactable_type,
+    (
+      COALESCE(a.user_id::CHAR, a.id::CHAR) || 
+      '_' ||
+      a.action ||
+      '_' ||
+      COALESCE(a.payload->>'reactable_id', '')
+    ) AS reaction_id
```

Comment:
This `reaction_id` is used to de-dupe reactions by the same person on the same reactable. E.g. a person liking, removing their like, and liking again- using this `reaction_id` we will only count one reaction in this situation.

The only exception is that you can like something, remove your like, and instead dislike something- this will still be counted as two reactions.